packages/common/components/blocks/company-details.marko

```diff
@@ -12,12 +14,13 @@ $ const fields = [
   { key: 'trainingInformation', label: 'Training', tag: 'p' },
   { key: 'servicesProvided', label: 'Other services', tag: 'p' },
 ];
+$ const lang = input.lang || 'en'
```

Comment:
Use double quotes in template files (bonus points for fixing the definitions above too)